modules/administration-guide/pages/networking-requirements.adoc

```diff
@@ -0,0 +1,40 @@
+:_content-type: CONCEPT
+:description: Networking requirements
+:keywords: administration guide, configuring, namespace, network policy, network policies, multitenant isolation
+:navtitle: Networking requirements
+:page-aliases: installation-guide:networking-requirements.adoc
+
+[id="networking-requirements_{context}"]
+= Networking Requirements
```

Comment:
Maybe other titles could be:
* Configuring Networking Requirements
* Configuring Network Policies

cc @ibuziuk

Reply:
Updated to `Configuring Network Policies`